admin/hotlist-service/src/test/java/io/mosip/hotlist/builder/RestRequestBuilderTest.java

```diff
@@ -85,7 +85,7 @@ public void testBuildRequest() throws HotlistAppException {
 		testRequest.setHeaders(null);
 	}
 
-	@Test(expected = HotlistAppException.class)
+	@Test(expected = Exception.class)
```

Comment:
Why the expected exception changed? we may have issue in code. please check.

Reply:
changed the exception.